deluge-labeler.py

```diff
@@ -136,87 +134,106 @@ def notmet_release(release):
         or "cbfm" in lowern
         or "eleanor" in lowern
         or "accomplishedyak" in lowern
-    ):
-        return True
-    else:
-        return False
+    )
 
 
-def limited_tracker(t_object, limited):
+def limited_tracker(t_object, limited = False):
     trackers = t_object[1].get("tracker", None)
     name = t_object[1].get("name", [None])
     label = t_object[1].get("label", [None])
 
-    if len(trackers) == 0:
+    if not trackers:
         return False
     if limited is True and label == "limiter":
-        return False
+        return False  # correct label already set
     if label == "sonarr" or label == "radarr":
-        return False
+        return False  # always ignore *arr-managed torrents
     if limited == 3:
-        if notmet_release(name) and label != "not-met" and label != "non-imported":
-            # print(f"{name} - {trackers} - {label}")
-            return True
-        else:
-            return False
+        # add 'not-met' label only for specific torrents whose label is not of {not-met,non-imported}:
+        return notmet_release(name) and label != "not-met" and label != "non-imported"
 
+    # from here on, only input value of limited = {True,False} values are processed:
     if valued_trackers(trackers):
         if limited is True:
-            return False
+            return False  # we don't want to set 'limiter' label if torrent is of valuable tracker because full BW should be allowed for them
         else:
-            if (notmet_release(name)) or (
+            # we don't want to set 'imported' label for some specific torrents OR if their label is of {not-met,imported,non-imported}
+            return not (notmet_release(name) or (
                 label == "not-met" or label == "imported" or label == "non-imported"
-            ):
-                return False
-            else:
-                return True
+            ))
 
+    # from here on we process limited = {True,False} of _non-valued_ trackers;
+    # we're deciding over setting either 'imported' OR 'limiter' labels:
     if label != "limiter" or label != "non-imported" or label != "not-met":
-        if "cross-seed" in label:
-            if valued_trackers(trackers):
+        # note in this block we _never_ allow for setting 'imported' label; is this expected?
+        if label.endswith(".cross-seed"):
```

Comment:
```suggestion
        if "cross-seed" in label:
```

My default data-based matching label is "cross-seed" - hence the usage before, it catches all variants since 'cross-seed' is never in any other label but one that needs to be managed.